5-app-infra/README.md

```diff
@@ -75,14 +76,14 @@ This Compute Engine instance is created using the base network from step `3-netw
 
 ### Deploying with Cloud Build
 
-1. Get the project ID from the previous procedure. Run the following in the
-   `4-projects/business_unit_1/shared` folder.
+1. Get the Infra Pipeline project ID from the previous step. Run the following in the
+   `gcp-projects/business_unit_1/shared` folder.
    ```
    terraform output cloudbuild_project_id
    ```
-1. Clone repo.
+1. Clone the policies repo. (This repo has the same name of the repo created in step 1-org but it is from a different project, you may need to rename the previous one to prevent a collision if you are cloning it in the same folder).
```

Comment:
Why aren't we using the same policy repo as the rest of the foundation? Is this so app pipeline can have a different set of rules?

Reply:
@rjerrems We are using the same policy files as the rest of the foundation, but not the same policy repository.

The infra pipeline has its own repositories, on for the code and one for the policies in the infra-pipeline project.

They can have different values but they start the same.

the infra-pipeline is independent from the main pipeline

Reply:
there is a issue to let the user use the artifact image created in 0-bootstrap.
we could open an issue to allow reuse of the policy repo too.

Reply:
Yeah I think we should do that, it will be a confusing customer experience to have to push the same policies to two different repos and create a greater opportunity for human error.